src/components/typeahead/typeahead.component.ts

```diff
@@ -99,13 +119,17 @@ export class TypeaheadComponent<T = any> implements OnChanges, OnDestroy {
     /** Emit the highlighted element when it changes */
     @Output() highlightedElementChange = new EventEmitter<HTMLElement>();
 
+    /** Emits when recently selected options change.*/
+    @Output() recentOptionsChange =  new EventEmitter<T[]>();

```

Comment:
Can we remove the additional space here after the `=`?

Reply:
Done